functions/sample_function.ts

```diff
@@ -31,12 +35,33 @@ export const SampleFunctionDefinition = DefineFunction({
   },
 });
 
+// This function takes the input from the open form step, adds formatting, saves our
+// updated object into the Slack hosted Datastore, and returns the updated message.
 export default SlackFunction(
   SampleFunctionDefinition,
-  ({ inputs }) => {
-    const { message } = inputs;
-    const updatedMsg =
-      `:wave: You submitted the following message: \n\n>${message}`;
+  async ({ inputs, client }) => {
+    const uuid = crypto.randomUUID();
+
+    // inputs.user is set from the interactivity_context defined in sample_trigger.ts
+    // https://api.slack.com/future/forms#add-interactivity
+    const updatedMsg = `:wave: ` + `<@${inputs.user}>` +
+      ` submitted the following message: \n\n>${inputs.message}`;
+
+    const sampleObject = {
+      original_msg: inputs.message,
+      updated_msg: updatedMsg,
+      object_id: uuid,
+    };
+
+    // Save the sample object to the datastore
+    // https://api.slack.com/future/datastores
+    await client.apps.datastore.put(
```

Comment:
nit: If you import `SampleObjectDatastore` you should be able to do `await client.apps.datastore.put<typeof SampleObjectDatastore>(` here

Reply:
<img width="1072" alt="Screen Shot 2022-10-03 at 9 32 27 AM" src="https://user-images.githubusercontent.com/10428517/193590354-552cdfac-53a2-4a44-a136-8c8ba8375eae.png">


Hmm - when I tried that, I kept getting this meesage

Reply:
Oh sorry, I think I may have been referencing some outdated content 🤦 , you'd need to access the definition via `await client.apps.datastore.put<typeof SampleObjectDatastore.definition>`

Reply:
oh - nice - that definitely worked. Thanks @shapirone  :)!